examples/sdk-snapgrid/src/components/MasonryGrid/MasonryGrid.module.css

```
@@ -0,0 +1,318 @@
+  column-gap: 16px;
+  /* No overflow:hidden here — lets break-inside work properly */
+}
+
+@media (max-width: 1200px) { .grid { columns: 3; } }
+@media (max-width: 820px)  { .grid { columns: 2; } }
+@media (max-width: 480px)  { .grid { columns: 2; column-gap: 10px; } }
+
+/* ── card ─────────────────────────────────────────────────── */
+.card {
+  break-inside: avoid;
+  page-break-inside: avoid; /* Safari */
```

Stylelint (17.9.0)

[error] Expected "page-break-inside" to be "break-inside" (property-no-deprecated)

(property-no-deprecated)